libsplinter/src/rest_api/actix_web_1/paging.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2019 Target Brands, Inc.
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+
+use crate::actix_web::{web, Error, HttpResponse};
+use crate::futures::{future::IntoFuture, Future};
+use crate::rest_api::{
+    paging::{PagingQuery, DEFAULT_LIMIT, DEFAULT_OFFSET},
+    ErrorResponse,
+};
+
+/// Return the PagingQuery from a Query object of key-value pairs.
+pub fn get_paging_query(
```

Comment:
Instead of implementing the parsing ourselves, maybe we can just do something like this: https://docs.rs/actix-web/1.0.9/actix_web/web/struct.Query.html#example

Reply:
We could do that as a future refactor or implementation - particularly in the actix3 work.

Reply:
Mainly, the reason to keep it as a map is due to the fact that most other areas were using it that way.